aiapy/calibrate/util.py

```diff
@@ -214,42 +243,57 @@ def get_pointing_table(start, end):
             table[c].unit = "arcsecond / pixel"
         if "INSTROT" in c:
             table[c].unit = "degree"
-    # Remove masking on columns with pointing parameters
     for c in table.colnames:
+        # Remove masking on columns with pointing parameters
         if any(n in c for n in ["X0", "Y0", "IMSCALE", "INSTROT"]) and hasattr(table[c], "mask"):
             table[c] = table[c].filled(np.nan)
     return table
 
 
-def get_error_table(error_table=None):
-    if error_table is None:
-        # This is to work around a parfive bug
-        # https://github.com/Cadair/parfive/issues/121
-        os.environ["PARFIVE_DISABLE_RANGE"] = "1"
-        error_table = fetch_error_table()
-        os.environ.pop("PARFIVE_DISABLE_RANGE")
-    if isinstance(error_table, str | pathlib.Path):
-        table = astropy.io.ascii.read(error_table)
-    elif isinstance(error_table, QTable):
-        table = error_table
+@manager.require("error_table_v3", *_URL_HASH_ERROR_TABLE[3])
+def _fetch_error_table_v3():
+    return manager.get("error_table_v3")
```

Comment:
Maybe do the same as with the correction tables. Name with "latest" and point the latest entry in the dictionary to whatever version is the latest?

Reply:
Fixed.